topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -0,0 +1,849 @@
+---
+layout: tutorial_hands_on
+
+title: "Galaxy Installation with Ansible"
+questions:
+- How does the Galaxy Ansible module work internally?
+- How can I install a Galaxy server with Ansible
+objectives:
+- Have an understanding of how Galaxy's Ansible roles are structured and interact with one another
+- Be able to use an Ansible playbook to install different flavors of Galaxy for different purposes
+time_estimation: "2h"
+key_points:
+- Basic deployment with Ansible is surprisingly easy
+- Complexity can grow over time as your organisation does, no need to start with playbooks like usegalaxy.org
+contributors:
+  - erasche
+requirements:
+  - type: "internal"
+    topic_name: admin
+    tutorials:
+      - ansible
+  -
+    title: "A server/VM to deploy Galaxy on"
+    type: "none"
+  - type: "external"
+    title: Ansible setup on your local machine
+    link: "https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html"
+  -
+    title: Comfort with Ansible roles and tasks
+    type: "none"
+---
+
+# Overview
+{:.no_toc}
+
+Now that you have some familiarity with Ansible and are comfortable running existing playbooks that you've written, we'll move on to installing Galaxy with a playbook.
```

Comment:
This intro is a bit too tailored for the Admin course, can you rephrase it so it applies also for people reading this on their own (and which may have learnt Ansible somewhere else)?

Reply:
This tutorial assumes the knowledge contained in the requirements, namely the existing ansible tutorial in the training material.

Reply:
I've seen that, what I don't like of the present formulation is the "Now that you have..." and "we'll move on" parts, which seem to assume you have just finished the Ansible tutorial, while the reader may have learnt it somewhere else or a long time ago.

What about the following:

```suggestion
This tutorial assumes you have some familiarity with Ansible and are comfortable with writing and running playbooks. Here we'll see how to install a Galaxy server using an Ansible playbook.
```

Reply:
Thanks for the suggestion, yeah, that fits for me :) Definitely needed to rework that formulation, don't want to put too much pressure on the reader, just that "there is an assumption/belief that you're comfortable with this already."